modules/programs/firefox.nix

```diff
@@ -97,7 +93,12 @@ let
         lib.concatStringsSep "\n"
         (map (itemToHTMLOrRecurse indentLevel) bookmarks);
 
-      bookmarkEntries = allItemsToHTML 1 bookmarks;
+      isToolbar = b: allItemsToHTML 1 (filter (bookmark: bookmark.toolbar == b) bookmarks);
```

Comment:
Use `lib.partition` instead of two passes of `filter`

Reply:
Done